tests/src/test/java/alluxio/hadoop/fs/AccumulatingReducer.java

```diff
@@ -59,6 +59,15 @@ public AccumulatingReducer() {
     LOG.info("Starting AccumulatingReducer on " + mHostname);
   }
 
+  /**
+   * This method accumulates values based on their type
```

Comment:
"Accumulates values based on their type."

(needs a period at the end of the sentence)